setup.py

```diff
@@ -15,33 +15,47 @@
     except IndexError:
         raise RuntimeError('Unable to determine version.')
 
-install_requires = [
-    'httptools>=0.0.9',
-    'ujson>=1.35',
-    'aiofiles>=0.3.0',
-    'websockets>=3.2',
-]
+setup_kwargs =  {
+        'name': 'sanic',
```

Comment:
@seemethere It would be much nicer to let Hound CI submit reviews to pull requests than looking into Travis CI logs.